static/css/components/reading-goal.less

```diff
@@ -1,17 +1,128 @@
 /* Styles for the yearly reading goal form */
 .goal-form {
   font-size: @font-size-title-medium;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  text-align: center;
+  width: 100%;
+  max-width: 100%;
+  box-sizing: border-box;
 
-  button {
-    display: inline-block;
-    width: fit-content;
-    margin-left: 10px;
+  &__question {
+    display: flex;
+    flex-flow: row wrap;
+    gap: 8px;
+    align-items: center;
+    justify-content: center;
+  }
+
+  &__learn-more {
+    font-size: @font-size-label-small;
+    color: @dark-grey;
+    text-decoration: none;
+  }
+
+  &__input-section {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    gap: 12px;
+    width: 100%;
+    box-sizing: border-box;
+    padding-right: 10px;
+  }
+
+  &__note {
+    margin: 0;
+    color: @dark-grey;
+    font-size: @font-size-label-small;
+    text-align: center;
+  }
+}
+
+.goal-form__question span {
+  font-weight: 500;
+}
+
+.goal-form__input-section input {
+  width: 80px;
+  padding: 8px 12px;
+  font-size: @font-size-label-large;
+  text-align: center;
+  border: 2px solid @lighter-grey;
+  border-radius: 4px;
+  outline: none;
+}
+
+.goal-form__input-section input:focus {
+  border-color: @primary-blue;
+  box-shadow: 0 0 0 2px fade(@primary-blue, 20%);
+}
+
+.goal-form__input-section button {
+  display: inline-block;
+  width: fit-content;
+  margin: 0;
+  padding: 9px 20px;
+  font-size: @font-size-label-large;
+}
+
+// Responsive design for smaller screens
+@media only screen and (max-width: @width-breakpoint-tablet) {
+  .goal-form {
+    gap: 12px;
+  }
+
+  .goal-form__question {
+    gap: 8px;
+    flex-wrap: wrap;
+  }
+
+  .goal-form__question span {
+    font-size: @font-size-label-medium;
+  }
+
+  .goal-form__input-section {
+    gap: 10px;
+    justify-content: center;
+    padding-right: 0;
+  }
+
+  .goal-form__input-section input {
+    width: 65px;
+    padding: 7px 8px;
+    font-size: @font-size-label-medium;
+  }
+
+  .goal-form__input-section button {
+    padding: 7px 14px;
+    font-size: @font-size-label-medium;
+  }
+}
+
+@media only screen and (max-width: 480px) {
```

Comment:
We can't have hard-coded breakpoints.  Please use one of [these three](https://github.com/internetarchive/openlibrary/blob/44b9fa1c99609991230193ec281e45efe4647db4/static/css/less/breakpoints.less#L3-L5)